### PR TITLE
Add illustrations for indicating explorer search states

### PR DESCRIPTION
**Pull request by arsengit**
- Created `IllustrationBlock` component
- Added `IllustrationConfig` config file
- Implemented `IllustrationBlock` in explorer pages to show their current state (pending, not requested, empty explorer, bad request)
- Deleted unused `svg` images
- Added `RequestStatusEnum`  enum

### CODE REVIEW

File: aim/web/ui/src/components/IllustrationBlock/IllustrationBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+function IllustrationBlock({
+  title,
+  content,
+  image,
+  page = 'metrics',
+  type = IllustrationsEnum.ExploreData,
+  className = '',
+  size = 'small',
+}: IIllustrationBlockProps): React.FunctionComponentElement<React.ReactNode> {
+  console.log(type, page);
```
Comment (roubkar) [inline aim/web/ui/src/components/IllustrationBlock/IllustrationBlock.tsx:25]: please remove `console.log`

File: aim/web/ui/src/components/MediaPanel/MediaPanel.tsx [hunk 2]
```diff
@@ -180,6 +182,8 @@ function MediaPanel({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  console.log(mediaType, illustrationType);
```
Comment (roubkar) [inline aim/web/ui/src/components/MediaPanel/MediaPanel.tsx:185]: please remove `console.log`

File: aim/web/ui/src/pages/Params/ParamsContainer.tsx [hunk 3]
```diff
@@ -110,6 +112,8 @@ function ParamsContainer(): React.FunctionComponentElement<React.ReactNode> {
     };
   }, []);
 
+  console.log(paramsData);
```
Comment (roubkar) [inline aim/web/ui/src/pages/Params/ParamsContainer.tsx:115]: please remove `console.log`

**Review by roubkar — CHANGES_REQUESTED** [review]

File: aim/web/ui/src/components/Table/Table.tsx [hunk 4]
```diff
@@ -894,7 +895,12 @@ const Table = React.forwardRef(function Table(
             />
           </div>
         ) : (
-          <EmptyComponent size='big' content={emptyText} />
+          <IllustrationBlock
+            page={illustrationConfig?.page || 'metrics'}
+            type={illustrationConfig?.type || IllustrationsEnum.EmptyData}
+            size={illustrationConfig?.size || 'xLarge'}
+            content={illustrationConfig?.type ? null : emptyText}
```
Comment (Hamik25) [inline aim/web/ui/src/components/Table/Table.tsx:902]: I think here you don't need pass content prop.

File: aim/web/ui/src/pages/Bookmarks/Bookmarks.tsx [hunk 5]
```diff
@@ -35,7 +37,12 @@ function Bookmarks({
               ))}
           </BusyLoaderWrapper>
           {!isLoading && data?.length === 0 ? (
-            <EmptyComponent size='big' content={'No Bookmarks'} />
+            <IllustrationBlock
+              size='xLarge'
+              page='bookmarks'
+              type={IllustrationsEnum.EmptyBookmarks}
+              content={'No Bookmarks Yet'}
```
Comment (Hamik25) [inline aim/web/ui/src/pages/Bookmarks/Bookmarks.tsx:44]: I think here you don't need pass content prop.

File: aim/web/ui/src/pages/Tags/TagDetail.tsx [hunk 6]
```diff
@@ -73,14 +73,15 @@ function TagDetail({
...
+
         <BusyLoaderWrapper
           isLoading={isRunsDataLoading}
           className='Tags__TagList__tagListBusyLoader'
         >
           {!isEmpty(tagRuns) ? (
             <TagRunsTable runsList={tagRuns} />
           ) : (
-            <EmptyComponent size='big' content='No Runs' />
+            <IllustrationBlock size='xLarge' content='No Runs' />
```
Comment (Hamik25) [inline aim/web/ui/src/pages/Tags/TagDetail.tsx:84]: I think here you don't need pass content prop.

**Review by Hamik25 — APPROVED** [review]

File: aim/web/ui/src/services/models/explorer/createAppModel.ts [hunk 7]
```diff
@@ -3177,9 +3321,17 @@ function createAppModel(appConfig: IAppInitialConfig) {
...
+                .call()
+                .then((data: IProjectParamsMetrics) => {
+                  model.setState({
+                    selectFormOptions: getParamsOptions(data),
+                  });
+                });
 
               model.setState({
-                requestIsPending: false,
+                // requestStatus: RequestStatusEnum.Ok,
```
Comment (KaroMourad) [inline aim/web/ui/src/services/models/explorer/createAppModel.ts:3334]: if we need this comment please add TODO here

File: aim/web/ui/src/types/services/models/metrics/metricsAppModel.d.ts [hunk 8]
```diff
@@ -17,6 +14,9 @@ import {
 } from 'utils/aggregateGroupData';
 import { AlignmentOptionsEnum } from 'utils/d3';
 
+import { IAppModelConfig, IGroupingConfig } from '../explorer/createAppModel';
+import { IImagesExploreAppConfig } from '../imagesExplore/imagesExploreAppModel';
```
Comment (KaroMourad) [inline aim/web/ui/src/types/services/models/metrics/metricsAppModel.d.ts:18]: lets change relative path to absolute

**Review by KaroMourad — APPROVED** [review]
lgtm

**Pull request merged**